library/std/src/io/mod.rs

```diff
@@ -2285,26 +2279,21 @@ pub trait BufRead: Read {
     /// // work with buffer
     /// println!("{buffer:?}");
     ///
-    /// // ensure the bytes we worked with aren't returned again later
+    /// // mark the bytes we worked with as read
     /// let length = buffer.len();
     /// stdin.consume(length);
     /// # std::io::Result::Ok(())
     /// ```
     #[stable(feature = "rust1", since = "1.0.0")]
     fn fill_buf(&mut self) -> Result<&[u8]>;
 
-    /// Tells this buffer that `amt` bytes have been consumed from the buffer,
-    /// so they should no longer be returned in calls to `read`.
+    /// Marks the given `amount` of additional bytes from the internal buffer as having been read.
+    /// Subsequent calls to `read` return bytes that have not yet been so marked.
```

Comment:
```suggestion
    /// Subsequent calls to `read` will not return bytes that have been marked as consumed.
```

Reply:
I'd really like to avoid phrasing this as a negative, and instead say what `read` will do as opposed to what it won't. Perhaps this would be better if "so marked" => "marked as read"?